okapi-common/src/main/java/org/folio/okapi/common/OkapiToken.java

```diff
@@ -41,16 +46,37 @@ private JsonObject getPayloadWithoutValidation() {
     }
   }
 
+  private String getFieldFromToken(String field) {
+    if (payloadWithoutValidation == null) {
+      return null;
+    }
+    return payloadWithoutValidation.getString(field);
+  }
+
   /**
    * Get the tenant out from the token.
    * Note there is no JWT validation taking place.
    * @return null if no token, or no tenant there
    */
   public String getTenant() {
```

Comment:
Taking a value out of a JWT token without validating the token can cause a security issue because anyone can spoof it.
Such a value must be handled with caution and care.
I suggest to rename the methods to draw developer attention to the missing validation:
* rename `getTenant()` to `getTenantWithoutValidation()`
* rename `getUsername()` to `getUsernameWithoutValidation()`
* rename `getUserId()` to `getUserIdWithoutValidation()`
* rename `getFieldFromToken(String)` to `getFieldFromTokenWithoutValidation(String)`

Reply:
fixed